django/views/generic/edit.py

```diff
@@ -163,6 +163,7 @@ def get_success_url(self):
         Returns the supplied URL.
         """
         if self.success_url:
+            self.success_url = force_text(self.success_url)
```

Comment:
should we include a comment that `force_text()`should be removed at the end of the deprecation period?